### ISSUE TITLE
Internal User: Move an Entity

### ISSUE THREAD

**Issue by andrea-williams** [bcgov/cas-registration#1926]
#### Dependencies

- Blocked by #1616 
- Blocking

#### Description:

Internal user's view of the "Move an Entity" form from within Report an Event/Change.

#### Acceptance Criteria:

Given I am an authenticated IRC user with the appropriate permissions
When I click the "Move an Entity" button
Then I am redirected to the Move an Entity form

Given I am an authenticated IRC user with the appropriate permissions
When I submit the Move an Entity form
Then the database is updated accordingly (taking into consideration the effective_date)

Wireframe: https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?m=dev&node-id=13836-1995&t=vFCjJSGep5uyNp9X-1

#### Development Checklist:

- [ ] Checklist item
- [ ] Checklist item
- [ ] Checklist item
- [ ] Meets the DOD

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

·**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

**Comment by zoeyli-46** [comment]
Hey @andrea-williams, I just saw this ticket which seems to be the same as #1945  
I actually just created #1949 which has all the form pages (3) for the internal user to move the entity. Could we move some items from this ticket and 1945 to the 1949, or maybe we can repurpose them so they don't overlap with #1949?

**Comment by andrea-williams** [comment]
Closing as duplicate
